torchcfm/conditional_flow_matching.py

```diff
@@ -49,7 +68,7 @@ def compute_mu_t(self, x0, x1, t):
             represents the source minibatch
         x1 : Tensor, shape (bs, dim)
             represents the source minibatch
-        t : float, shape (bs, 1)
```

Comment:
What do you think about `t : Tensor, shape (bs)`? 

Also x1 can technically be multi-dimensional aka (bs, *dims). Channels, Height, Width for images, etc.

Reply:
Good catch. I suggest `t : FloatTensor, shape (bs)`

Fair point. I will update x's shape as well